### ISSUE TITLE
Rejections from Apple

### ISSUE THREAD

**Issue by dimovdaniel** [chirag04/react-native-in-app-utils#125]
Great module but I'm experiencing the following problem. 

My in app product (recurring subscription) are not yet clear for sale. Their status is  "Waiting for review" - for days.
And while testing I'm getting "invalid_product".  

I submitted app like this, but apple rejected, saying that app has bugs "invalid_product". 

So what is the procedure, to make my in app product clear for sale

**Comment by chirag04** [comment]
any item that is not cleared for sale will return invalid product. you should wait for it to be cleared. Make sure you attached screenshots etc to the in app purchase.

let me know if this is still a problem.

**Comment by dimovdaniel** [comment]
I have removed inapp purchase for now, and submited the app with out them. 
Later when app is on app store, will add them back. 

No matter what I did , my inapp purchases where rejected, because the app was displaying "invalid_product". 

So I was like in a loop. My inapp purchases where "Waiting for review" and my app was displaying "invalid_product" and it was rejected. 

Will see how it will go this way. 
Thanks for the reply.
